src/app/components/challenge/challengephases/phasecard/phasecard.component.ts

```diff
@@ -26,17 +29,36 @@ export class PhasecardComponent implements OnInit {
    */
   endDate: any;
```

Comment:
To maintain consistency, `endDate` could be a `Date` or `String` as it was in other files.